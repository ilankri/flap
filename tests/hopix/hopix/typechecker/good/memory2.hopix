val r : cell (int) = ref (0)

val write : unit =
  r := 42

val read : int =
  !r