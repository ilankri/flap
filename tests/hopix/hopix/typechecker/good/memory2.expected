r : cell(int)
write : unit
read : int
